explorer/client/src/index.tsx

```diff
@@ -9,7 +9,6 @@ import { BrowserRouter as Router } from 'react-router-dom';
 
 import App from './app/App';
 import { CURRENT_ENV, IS_PROD_ENV, IS_STAGING_ENV } from './utils/envUtil';
-import reportWebVitals from './utils/reportWebVitals';
```

Comment:
I removed this because we didn't actually log the web vitals anywhere. We can always re-introduce this in the future.